src/core/Akka/Event/LoggingBus.cs

```diff
@@ -106,14 +107,29 @@ internal void StartDefaultLoggers(ActorSystemImpl system)
                     shouldRemoveStandardOutLogger = false;
                     continue;
                 }
-                
-                try
+
+                if (asyncStart)
```

Comment:
So my gut tells me:

1. Either get rid of this setting and just make async starts the default OR
2. Make it so this setting is opt-in - i.e. it should default to synchronous.

That way, in the event of breaking behavior, it can't catch users by surprise - it only happens when you explicitly opt in to changing this setting.

What do you think?

Reply:
Agree, let's disable by defailt